blocks/identity-block/themes/news.json

```diff
@@ -313,6 +313,60 @@
 			}
 		}
 	},
+	"login-form-sign-up-div": {
+		"styles": {
+			"default": {
+				"display": "block",
+				"text-align": "center",
+				"width": "100%",
+				"margin": 0,
+				"line-height": "150%",
+				"font-size": "1rem",
+				"margin-bottom": "var(--global-spacing-5)",
+				"components": {
+					"paragraph": {
+						"font-family": "Inter",
```

Comment:
Is there a reason this font-family is hard coded to Inter and not using `"var(--font-family-primary)"` (which defaults to Inter out of the box)? I'm seeing this in a few other places too.